CONTRIBUTORS.rst

```diff
@@ -0,0 +1,20 @@
+.. _contributors:
+
+Contributors
+------------
+
+We follow the |icon| `all-contributors`_ specification.
+Contributions of any kind to ``pyvista`` are welcome!
+
+It's important that we recognise and acknowledge **all** our contributors, not just the ones who push code.
+So no matter how large or small, we're grateful for your contribution and caring enough to make a difference.
+
+- Maksym Kalaidov (`@mkskalai <https://github.com/mkskalai>`_) (Promotion)
```

Comment:
Did you confirm with them that they are happy to be added here? Perhaps it would be a good rule of thumb to always ping the people on such PRs :) (When they have github users, at least.)